### DISCUSSION TITLE
PHP8 Requirement

### DISCUSSION THREAD

**Discussion by Morpheus1822** [firefly-iii/firefly-iii#5105] [General]
Hey everyone,
I was looking for the reason why with the recent 5.6.0 release php8 is a requirement as stated in the release notes. But I didn't find anything in the docs (but perhaps I looked at the wrong place).

I'm hosting Firefly on a Debian host and with the very recent Bullseye upgrade, the default php version on Debian is 7.4 (as is in the current LTS version of Ubuntu). I'm very reluctant to add 3rd party packages to my production servers, which makes it difficult to install php8.

So I'm wondering if it is possible to upgrade to 5.6.0 (and later) without php8. And am I the only one, who is a bit unhappy with this requirement?

**Comment by JC5** [comment]
All I can do is apologize about the bad communication. The release notes are now up to date and the documentation is as well. It will be a moment before the CDN updates all pages.

**Reply by hoangduy1505** [reply]
Hi James, can you add version 5.11 to docker. I am using nas synology to run docker, now my nas not yet support php 8.0 (max current version is 7.4)

**Reply by JC5** [reply]
The latest version that accepts php 7.4 is 5.5.13.
